### PR TITLE
Fixing last name count logic for database loading

### PR DESCRIPTION
**Pull request by priyankasprakash**

### CODE REVIEW

**Review by uttampawar — COMMENTED** [review]
@priyankasprakash Has this been tested? Specifically the config.json and runspec.py changes. You should have made two separate PR's. One for client and another for Server.
Did you get a chance to do a long run? How does result looks like for the long run?

File: Node-DC-EIS-cluster/controllers/loader.js [hunk 1]
```diff
@@ -171,6 +171,19 @@ exports.initDb = function initDB(req, res) {
   var e_count=0;
   var zipcode_current_index=0;
   var lastname_current_index=0;
+  var original_lname_len=lnames.length;
+  if (lnames.length < lastnamecount) {
```
Comment (egalli) [inline Node-DC-EIS-cluster/controllers/loader.js]: I think it should be <= instead of <

File: Node-DC-EIS-cluster/controllers/loader.js [hunk 2]
```diff
@@ -171,6 +171,19 @@ exports.initDb = function initDB(req, res) {
...
+    var new_namecount = lastnamecount - lnames.length;
+    var idx = 0;
+    for (var ii=0; ii < new_namecount; ii++) {
+      var new_name = lnames[idx]+ii.toString();
```
Comment (egalli) [inline Node-DC-EIS-cluster/controllers/loader.js]: .toString() is redundant. JS automatically converts Numbers to Strings when concatenating them to a String

**Review by egalli — CHANGES_REQUESTED** [review]

File: Node-DC-EIS-cluster/controllers/loader.js (hunk 2 above)
Comment (uttampawar) [reply Node-DC-EIS-cluster/controllers/loader.js]: @egalli @priyankasprakash Even though JS converts number to strings automatically, I would rather have a clean use of API. Use of Number.toString() API is a clean way to use it.

File: Node-DC-EIS-cluster/controllers/loader.js (hunk 1 above)
Comment (uttampawar) [reply Node-DC-EIS-cluster/controllers/loader.js]: Agree.

**Comment by uttampawar** [comment]
LGTM.

**Pull request merged**